src/rpc/format/cbor/convert.rs

```diff
@@ -54,23 +58,71 @@ impl TryFrom<Cbor> for Value {
 			Data::Tag(t, v) => {
 				match t {
 					// A literal datetime
-					TAG_DATETIME => match *v {
+					TAG_STRING_DATETIME => match *v {
 						Data::Text(v) => match Datetime::try_from(v) {
 							Ok(v) => Ok(v.into()),
 							_ => Err("Expected a valid Datetime value"),
 						},
 						_ => Err("Expected a CBOR text data type"),
 					},
+					// An epoch-based datetime (https://www.rfc-editor.org/rfc/rfc8949.html#name-epoch-based-date-time)
+					TAG_EPOCH_DATETIME => match *v {
+						Data::Integer(v) => match Datetime::try_from(i128::from(v)) {
+							Ok(v) => Ok(v.into()),
+							_ => Err("Expected a valid Datetime value"),
+						},
+						Data::Float(v) => match Datetime::try_from(v) {
+							Ok(v) => Ok(v.into()),
+							_ => Err("Expected a valid Datetime value"),
+						},
```

Comment:
I don't want to have support for datetimes which can drop precision. This will lead to errors down the line where users presume that there are issues with the database (if they even notice the loss of precision), and not with the CBOR implementation they are using. We can still support `TAG_EPOCH_DATETIME` as seconds, but not floating point numbers.

Reply:
@Odonno let's not do tag 1 at all, I don't want to have a half-implemented spec